tests/sanity/ignore-2.10.txt

```diff
@@ -0,0 +1,5 @@
+plugins/action/iosxr.py action-plugin-docs
+plugins/modules/iosxr_command.py validate-modules:doc-missing-type
```

Comment:
Can you add a comment on why we need these ignores for 2.10?
Afaiu, we would just need to ignore the `action-plugin-docs`.

Reply:
yes, you are right. extra ignore have been removed now. Thanks

Reply:
Done